ROS/Chapter_6.ipynb

```diff
@@ -0,0 +1,1208 @@
+{
```

Comment:
<p>pass <code>round_to=2</code> to <code>az.summary</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/Bambi_resources/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>